src/Networks/Blockcore.Networks.SeniorBlockCoin/Networks/Policies/SeniorBlockCoinStandardScriptsRegistry.cs

```diff
@@ -0,0 +1,64 @@
+using System.Collections.Generic;
+using System.Linq;
+using Blockcore.Consensus.ScriptInfo;
+using Blockcore.Consensus.TransactionInfo;
+using Blockcore.Networks;
+using NBitcoin.BitcoinCore;
+
+namespace Blockcore.Networks.SeniorBlockCoin.Networks.Policies
+{
+   /// <summary>
+   /// Blockcore sample coin-specific standard transaction definitions.
+   /// </summary>
+   public class SeniorBlockCoinStandardScriptsRegistry : StandardScriptsRegistry
+   {
+      // See MAX_OP_RETURN_RELAY in stratisX, <script.h>
+      public const int MaxOpReturnRelay = 40;
```

Comment:
This should probabl be "8x" something, not 40. Or do you want to support only a single signature?

Reply:
This was the default value. I change it to 83.

Reply:
Great!